tests/protos_append/builder_ext.rs

```diff
@@ -185,3 +185,67 @@ impl OrderByBuilder {
         builder
     }
 }
+
+impl RecommendPointsBuilder {
+    pub fn new(
+        collection_name: impl Into<String>,
+        positive: impl Into<Vec<PointId>>,
+        negative: impl Into<Vec<PointId>>,
+        positive_vectors: impl Into<Vec<Vector>>,
+        negative_vectors: impl Into<Vec<Vector>>,
```

Comment:
I _think_ some of these may be optional/empty. In that case we may want to prefer making them properly optional through the builder, defaulting to an empty vector.

cc @coszio

Reply:
Yeah, [here](https://qdrant.tech/documentation/concepts/explore/#using-only-negative-examples) we also describe it may be beneficial to use only negative vectors.

Also, `positive` and `positive_vectors` likely are not used together. That's another reason for making them optional.

So to rephrase; I'd recommend to make these 4 parameters optional, defaulting to an empty list (if needed).

Reply:
Resolved in e8973a7a47dbd45779ee739635e9e176e462a8bf

Reply:
> I think some of these may be optional/empty. In that case we may want to prefer making them properly optional through the builder, defaulting to an empty vector.

Yeah, both negatives and positives should be optional. The server will decide if that is valid when checking the recommend `strategy`

I have an idea to improve UX, though. In the `with_positive`/`positive` and `with_negative`/`negative` methods we can accept a `Into<RecommendExample>` which will route it to the correct list

Reply:
Yeah, second thing is what I tried to say!